app/models/payment.rb

```diff
@@ -0,0 +1,55 @@
+class Payment < ApplicationRecord
+  def self.create_instance(cal_amount, cal_interest, cal_pmt, cal_principal, cal_day, cal_end_balance)
```

Comment:
This method is not necessary at all. It's doing exactly the same as `new`.
```
Payment.new(start_balance: cal_amount,...)
```
Calling this 🔼 , it's better that 
```
Payment. create_instance(1435, 23...)
```
Because it's the same